test/runtest.py

```diff
@@ -1511,6 +1515,20 @@ def wait_for_object_table():
 
     ray.worker.cleanup()
 
+    ray.worker._init(redirect_output=True, num_local_schedulers=2, start_ray_local=True)
+
+    @ray.remote
+    def say_hi():
+      print("hi")
+
+    say_hi.remote()
+    log_table = ray.global_state.log_table()
+    print(log_table)
+    self.assertEqual(isinstance(log_table[node_ip_address], dict), True)
+
```

Comment:
We should probably check that `"hi"` appears in the log info.

Also, it might not actually be there yet when you check, so if it isn't then maybe we should loop until it is (so that the test doesn't fail sometimes). We do this for some other tests in this file, e.g., `testGlobalStateAPI`.